content/docs/reference/config/istio.mesh.v1alpha1/index.html

```diff
@@ -75,7 +75,7 @@ <h2 id="ConfigSource">ConfigSource</h2>
 </tr>
 <tr id="ConfigSource-tls_settings">
 <td><code>tlsSettings</code></td>
-<td><code><a href="/docs/reference/config/istio.networking.v1alpha3.html#TLSSettings">istio.networking.v1alpha3.TLSSettings</a></code></td>
+<td><code><a href="/docs/reference/config/networking/v1alpha3/virtual-service.html#TLSSettings">istio.networking.v1alpha3.TLSSettings</a></code></td>
```

Comment:
This is a generated link that the generator is messing up. TLSSettings is in destination-rule.html, but the generator seems to have arbitrarily picked virtual-service.html instead.

@geeknoid Do you know if there is a way to annotate or otherwise fix the generator here?

Reply:
This is exactly what I was warning @rshriram about links between these protos. He keeps just dismissing it with "we don't have links across protos".

Support for "one HTML file per proto" was added to the proto extraction tool by someone at Google that needed that feature for another project. I was never convinced the change would deal with cross-proto links correctly, and here's evidence.

I'll take a look at the tool to see if this is an easy fix.

Reply:
@rshriram 

I've updated the proto doc extraction tool in istio/tools and have submitted https://github.com/istio/api/pull/812 to start using there. If the original PR in istio/api is retried using this latest version of the tool, it should fix these types of link problem.